From 64a4f5d5990fa7e2210d232ab0df174d45478b0d Mon Sep 17 00:00:00 2001
From: Igal Zeltser <izeltser@habana.ai>
Date: Tue, 06 Dec 2022 16:59:22 +0200
Subject: [PATCH] [SW-0] pif_arc: rename delay function

Delay function for pif arc is renamed to better reflect the time units

Change-Id: I42b0e6845faade03883faf7dd1b29b8ae6affd5a
Signed-off-by: Igal Zeltser <izeltser@habana.ai>
---

diff --git a/app/pcie_pif/pcie_pif.c b/app/pcie_pif/pcie_pif.c
index e30c89e..e4c34c1 100644
--- a/app/pcie_pif/pcie_pif.c
+++ b/app/pcie_pif/pcie_pif.c
@@ -86,7 +86,7 @@
 static void flag_flr_done(void)
 {
 	regmap_writel(1, pif_arc.pf_flr_done);
-	pif_ts_delay(FLR_ACTIVE_SIG_WIDTH_US);
+	pif_ts_delay_us(FLR_ACTIVE_SIG_WIDTH_US);
 	regmap_writel(0, pif_arc.pf_flr_done);
 	hl_pif_report_sts(PIF_STAT_FLR_DONE_REPORTED);
 }
@@ -113,7 +113,7 @@
 static void set_flr_active(void)
 {
 	regmap_writel(1, pif_arc.pif2psoc_flr_active);
-	pif_ts_delay(FLR_ACTIVE_SIG_WIDTH_US);
+	pif_ts_delay_us(FLR_ACTIVE_SIG_WIDTH_US);
 	regmap_writel(0, pif_arc.pif2psoc_flr_active);
 	hl_pif_report_sts(PIF_STAT_INT_RESET_ACTIVE);
 }
diff --git a/arch/gaudi3/arc/common.c b/arch/gaudi3/arc/common.c
index 59180df..22f5c4a 100644
--- a/arch/gaudi3/arc/common.c
+++ b/arch/gaudi3/arc/common.c
@@ -850,7 +850,7 @@
 	regmap_writel(FW_WORD_NUM, mmD0_PCIE_WRAP_PHY_FW_FSM_SIZE);
 
 	/* let PHY_FW_FSM_SIZE propagate */
-	pif_ts_delay(1);
+	pif_ts_delay_us(1);
 
 	/*
 	 * PHY FW FSM EN and Direct FW write mechanism enable
diff --git a/lib/common/pif_ts.c b/lib/common/pif_ts.c
index 80f2386..df040ff 100644
--- a/lib/common/pif_ts.c
+++ b/lib/common/pif_ts.c
@@ -47,7 +47,7 @@
 /* pif timestamp counter doesn't roll over when max value is reached */
 #define PIF_TIMESTAMP_MAX_VAL	0xFFFFFFFF
 
-void pif_ts_delay(u32 usec)
+void pif_ts_delay_us(u32 usec)
 {
 	u32 start, current, cycle;
 	u64 end;
diff --git a/lib/common/pif_ts.h b/lib/common/pif_ts.h
index 9a44ec1..1570ab0 100644
--- a/lib/common/pif_ts.h
+++ b/lib/common/pif_ts.h
@@ -37,7 +37,7 @@
 bool pif_ts_is_rearm_enable(void);
 
 /* delay execution for specified time (in microseconds) */
-void pif_ts_delay(u32 usec);
+void pif_ts_delay_us(u32 usec);
 
 /*
  * Check if it's possible to meassure delay_usec timeout from start_usec
